src/hotspot/share/opto/vectornode.hpp

```diff
@@ -93,7 +93,7 @@ class VectorNode : public TypeNode {
   static bool is_type_transition_short_to_int(Node* n);
   static bool is_type_transition_to_int(Node* n);
```

Comment:
Not sure if we should add these 2 l2i candidates into `is_type_transition_to_int`. As they are indeed transformed to int.